docs/configuration/environment-checks.md

```diff
@@ -1,33 +0,0 @@
-# Environment Checks
```

Comment:
@jeremydmiller, something went really wrong with docs.

Reply:
Getting really bored with how the markdown files get massacred by big changes